service/backup/utils.go

```diff
@@ -116,6 +131,53 @@ func newServiceAccountDefinition(backup *backupv1alpha1.Backup, config config) s
 	}
 }
 
+func (b *backupRunner) getReplicaSets() []extensionsv1.ReplicaSet {
+	tmp := []extensionsv1.ReplicaSet{}
+	replicas := int32(1)
+
+	templates, err := b.BaasCLI.Appuio().PreBackupPods(b.backup.GetNamespace()).List(metav1.ListOptions{})
+
+	if err != nil {
+		b.Logger.Errorf("could not list podtemplates: %v", err)
+	} else {
+		for _, template := range templates.Items {
+
+			template.Spec.Pod.PodTemplateSpec.ObjectMeta.Annotations = map[string]string{
+				b.config.backupCommandAnnotation: template.Spec.BackupCommand,
+				b.config.fileExtensionAnnotation: template.Spec.FileExtension,
+			}
+
+			podLabels := map[string]string{
+				"backupCommandPod": "true",
```

Comment:
There should be a label which uniquely defines a pod, if multiple PreBackupPods are defined.